client/state/sites/plans/selectors.js

```diff
@@ -98,6 +118,32 @@ export function getPlanDiscountedRawPrice(
 }
 
 /**
+ * Returns a plan price before discount
+ *
+ * @param  {Object}  state         global state
+ * @param  {Number}  siteId        the site id
+ * @param  {String}  productSlug   the plan product slug
+ * @param  {Boolean} isMonthly     if true, returns monthly price
+ * @return {Number}                plan raw price
+ */
+export function getPlanRawPrice(
```

Comment:
Sounds good. I'll add the site prefix for the two new ones I'm adding. We can rename the other ones in follow up janitorials